src/plugins/vis_type_timeseries/public/application/components/yes_no.tsx

```diff
@@ -9,17 +9,18 @@
 import React, { useCallback } from 'react';
 import { EuiRadio, htmlIdGenerator } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
-import { TimeseriesVisParams } from '../../types';
+import type { TimeseriesVisParams } from '../../types';
+import type { Annotation } from '../../../common/types';
 
-interface YesNoProps<ParamName extends keyof TimeseriesVisParams> {
+interface YesNoProps<ParamName extends keyof TimeseriesVisParams | keyof Annotation> {
```

Comment:
wrong type. not sure that YesNo component should know anything about Annotation.

Reply:
and TimeseriesVisParams. Looks like bad typed Component. Can we use string instead?